src/data/schema/admin_setting.sql

```diff
@@ -4,4 +4,4 @@ create table admin_settings(
 );
 
 INSERT INTO admin_settings (semester)
-VALUES (' ');
+VALUES ('SUMMER 2020');
```

Comment:
We should not have a default that is RPI specific or time specific in table definitions, this must be extracted out to the get semester default. 
One solution could be to check if return value is Null from query, then select the first semester that is found from the list of classes.